### PR TITLE
Renaming 1.1 Introduction to Prerequisites or similar

### LINKED ISSUE #134: Improve the title 1.1 Introduction in Getting Started
The title **1.1 Introduction** does not look like introductory content. It looks more like prerequisites or requirements. The page talks about what EOSIO versions you need, the development experience you need, operating system, development tools, IDE, etc. 

Suggest changing the title to **Prerequisites** or something similar. 

Need the team's inputs. 

@dskvr @iamveritas @dixia @lparisc

### PR DESCRIPTION
**Pull request by bobgt**
Resolves #134 

Suggestions to improve the title 1.1 Introduction based on the nature of the content.

### CODE REVIEW

File: docs/02_getting-started/02_development-environment/02_introduction.md [hunk 1]
```diff
@@ -44,27 +36,39 @@ The EOSIO software supports the following environments for development and/or de
...
 - [Sublime Text](https://www.sublimetext.com/)
 - [Atom Editor](https://atom.io/)
 - [CLion](https://www.jetbrains.com/clion/)
 - [Eclipse](http://www.eclipse.org/downloads/packages/release/oxygen/1a/eclipse-ide-cc-developers)
 - [Visual Studio Code](https://code.visualstudio.com/)
 
+[[info]]
+| The resources listed above are developed, offered, and maintained by third-parties and not by block.one. Providing information, material, or commentaries about such third-party resources does not mean we endorse or recommend any of these resources. We are not responsible, and disclaim any responsibility or liability, for your use of or reliance on any of these resources. Third-party resources may be updated, changed or terminated at any time, so the information below may be out of date or inaccurate. USAGE AND RELIANCE IS ENTIRELY AT YOUR OWN RISK
```
Comment (dixia) [inline docs/02_getting-started/02_development-environment/02_introduction.md]: this callout needs to cover EOS Studio as well

File: docs/02_getting-started/02_development-environment/02_introduction.md [hunk 2]
```diff
@@ -1,37 +1,29 @@
 ---
-content_title: "1.1: Introduction"
-link_text: "1.1: Introduction"
+content_title: "1.1: Prerequisites"
```
Comment (dixia) [inline docs/02_getting-started/02_development-environment/02_introduction.md]: This is not an introduction for eosio but also not only contains prerequisite. It is more of an introduction to this tutorial.

**Comment by dskvr** [comment]
Depends on https://github.com/EOSIO/thorka/issues/243

**Comment by iamveritas** [comment]
this has been addressed in a different PR

**Pull request closed without merging**